firmware/spade/src/shared/FatFs_SPI/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+add_library(FatFs_SPI INTERFACE)
+target_sources(FatFs_SPI INTERFACE
```

Comment:
hmm I think it might be effective to use globs here

Reply:
i'm not sure if this is necessary - we could use globs, but from what I can see you would have to do something with `file()` and it might just be not as simple especially since `ff15/source/diskio.c` is specifically left out and spade will not compile if it is included